src/languageSupport/languages/flux/lsp/EXPLAINER.md

```diff
@@ -0,0 +1,64 @@
+# LSP connection to the monaco-editor
```

Comment:
This md file will be deleted before merge.
Left in only for now -- to answered anticipated questions (in case anyone else is reviewing the docs for these monaco-editor related libraries.)

Reply:
deleted.